### ISSUE TITLE
Recommend updates to istio-cni install instructions

### ISSUE THREAD

**Issue by bordenit** [rancher/rke2#3830]
Istio-cni does not work [per these instructions](https://ranchermanager.docs.rancher.com/v2.6/integrations-in-rancher/istio/configuration-options/install-istio-on-rke2-cluster) on RHEL 7 without additional file. Had to add istio-iptables.conf to get it to work.

See also https://github.com/istio/istio/issues/23009

content of /etc/modules-load.d/istio-iptables.conf
br_netfilter
nf_nat
xt_REDIRECT
xt_owner
iptable_nat
iptable_mangle
iptable_filter

**Comment by brandond** [comment]
The docs you linked to are for Rancher 2.6, not for RKE2. The RKE2 docs are at https://docs.rke2.io/known_issues#istio-in-selinux-enforcing-system-fails-by-default. I can transfer this issue to the correct repo if you'd like, or you can open another issue on that side.

**Comment by bordenit** [comment]
Yes, feel free to transfer, and thanks for the good link. Hopefully, they just remove it from the other place. Google's first stop is that Rancher page.
![image](https://user-images.githubusercontent.com/49888529/214147414-6a83a86d-6114-4966-8136-e9d29e2e98f7.png)

**Comment by brandond** [comment]
RKE2 doesn't ship with Istio, so we don't cover it extensively in our docs. Rancher does include a chart for it, which is why it is covered in their docs.
